jsonnet/prometheus-operator/admission-webhook.libsonnet

```
@@ -0,0 +1,123 @@
+local defaults = {
+  local defaults = self,
+  name: 'prometheus-operator-admission-webhook',
+  namespace: error 'must provide namespace',
+  version: error 'must provide version',
+  admissionWebhookImage: error 'must provide admissionWebhookImage',
+  port: 8443,
+  replicas: 2,
+  resources: {
+    limits: { cpu: '200m', memory: '200Mi' },
+    requests: { cpu: '50m', memory: '50Mi' },
+  },
+  commonLabels:: {
+    'app.kubernetes.io/name': defaults.name,
+    'app.kubernetes.io/version': defaults.version,
+  },
+  selectorLabels:: {
+    [labelName]: defaults.commonLabels[labelName]
+    for labelName in std.objectFields(defaults.commonLabels)
+    if !std.setMember(labelName, ['app.kubernetes.io/version'])
+  },
+};
+
+
+function(params) {
+  local aw = self,
+  config:: defaults + params,
```

```suggestion
  _config:: defaults + params,
```

Let's keep convention from kube-prometheus and prefix hidden variable with underscore.